### PR TITLE
[NUI][Overview] Added pallete api link to nui overview file.

### PR DESCRIPTION
**Pull request by mihashco**
### Change Description ###
- Added missing link to nui/overview.md

### CODE REVIEW

**Comment by mijin85cho** [comment]
@sagarkedari As @annie-samsung is absent, I inform you this PR instead. Would you please assign who will start reviewing this PR instead?

File: docs/application/dotnet/guides/user-interface/nui/overview.md [hunk 1]
```diff
@@ -13,31 +13,35 @@ After you have set up the NUI development environment, you can quickly create ri
...
+  Provides a mechanism for overlaying groups of views on top of each other.
 
-- [UI Components](ui-components.md)
+- [Layouts](layouts.md)
 
-  Interactive components for layout and user interface, such as buttons, table view, text controls, image view, flex container, slider, and video view.
+  Provides advanced reusable laying out capabilities.
 
-- [Animation](animation.md)
+- [Palette](palette.md)
```
Comment (asm-foysal) [inline docs/application/dotnet/guides/user-interface/nui/overview.md:32]: Although I can see "Palette" on the left navigation panel in the overview page, once I click the hyperlink, the file does not show up in the left navigation panel anymore.

**Overview screenshot:**
image

**Screenshot after clicking on hyperlink:**
image

**Review by asm-foysal — COMMENTED** [review]
@mihashco there's a problem with the "Palette" file, it's not showing up in the ToC once I click it. Perhaps the problem is due to the changed file name.

**Review by asm-foysal — COMMENTED** [review]
@mihashco thank you for addressing the ToC issue. I've nothing further to add to this PR.

@mijin85cho I'm done reviewing this PR, before I approve it, I'd request you to give it a quick look and see if everything is in order.

**Comment by asm-foysal** [comment]
@mijin85cho everything is in order with this PR, should I go ahead and approve it?

**Comment by mijin85cho** [comment]
Hello, @asm-foysal 
Yes this PR seems OK to go. Would you please proceed the process?

**Review by asm-foysal — APPROVED** [review]
@mihashco I'm approving this PR, thank you for your patience.

@mijin85cho this PR is ready to be merged.

**Pull request merged**